Lib/idlelib/config-extensions.def

```diff
@@ -32,68 +52,14 @@
 # See config-keys.def for notes on specifying keys and extend.txt for
 # information on creating IDLE extensions.
 
-[AutoComplete]
-enable=True
-popupwait=2000
-[AutoComplete_cfgBindings]
-force-open-completions=<Control-Key-space>
-[AutoComplete_bindings]
-autocomplete=<Key-Tab>
```

Comment:
The '_bindings' entries had two effects:
1. event_add a virtual event associated with the event sequences.
2. bind the virtual event to an event handler (named after the event).
This new line in EditorWindow did the second:
`        text.bind("<<autocomplete>>", autocomplete.autocomplete_event)`
The lack of a replacement for 1. disabled the use of tab to open and tab tab to close a completion box.  Adding
`            self.text.event_add('<<autocomplete>>', '<Key-Tab>')`
in AutoComplete.__init__ fixed the problem.  I am thinking that both parts should be in the same file.  Either the event_add should be in EditorWindow or the binding should be in AutoComplete.  I am leaning toward the latter, but will decide tomorrow.